packages/redis-v5/lib/shared.js

```diff
@@ -98,7 +98,7 @@ module.exports = (context, heroku) => {
 
           redis.app = db.addon.app
           redis.config_vars = db.addon.config_vars
-          const { formation, metaas_source, port, ...filteredRedis } = redis
+          const { formation, meta_as_source, port, ...filteredRedis } = redis
```

Comment:
Not sure what this is doing. 
https://cs.github.com/?scopeName=All+repos&scope=&q=metaas_source
shows 5 hits that more or less link back to this thing. It's probably an API.

I left it here so that I could ask "what is this?"